test/dotnet-new.Tests/GivenThatIWantANewCSApp.cs

```diff
@@ -35,6 +35,24 @@ public void When_NewtonsoftJson_dependency_added_Then_project_restores_and_runs(
                 .Should().Pass();
         }
         
+        [Fact]
+        public void When_dotnet_build_is_invoked_Then_project_builds_without_warnings()
+        {
+            var rootPath = Temp.CreateDirectory().Path;
+
+            new TestCommand("dotnet") { WorkingDirectory = rootPath }
+                .Execute("new");
+
+            new TestCommand("dotnet") { WorkingDirectory = rootPath }
+                .Execute("restore");
+
+            var buildResult = new TestCommand("dotnet") { WorkingDirectory = rootPath }
+                .ExecuteWithCapturedOutput("build");
+            
+            buildResult.Should().Pass();
```

Comment:
You can do all these assertions in one call using the "And" property.  For example:

```
            new BuildCommand(rootPath)
                .ExecuteWithCapturedOutput()
                .Should()
                .Pass()
                .And
                .NotHaveStdErr();
```